homeassistant/components/uvc/camera.py

```diff
@@ -106,6 +107,20 @@ def motion_detection_enabled(self):
         caminfo = self._nvr.get_camera(self._uuid)
         return caminfo['recordingSettings']['motionRecordEnabled']
 
+    @property
+    def supported_features(self):
+        """Return supported features."""
+        return SUPPORT_STREAM
```

Comment:
This should check if RTSP is enabled as well.

Reply:
Added. 👍